web-api/src/documents/generateStampedCoversheetLambda.js

```diff
@@ -0,0 +1,21 @@
+const { genericHandler } = require('../genericHandler');
+
+/**
+ * used for generating a stamped coversheet
+ *
+ * @param {object} event the AWS event object
+ * @returns {Promise<*|undefined>} the api gateway response object containing the statusCode, body, and headers
+ */
+exports.generateStampedCoversheetLambda = event =>
+  genericHandler(
+    event,
+    async ({ applicationContext }) => {
+      await applicationContext
+        .getUseCases()
+        .generateStampedCoversheetInteractor(applicationContext, {
+          ...event.pathParameters,
```

Comment:
I would say that the proxy that is calling this endpoint should not send the docketNumber or docketEntryId over in the post body.  Right now I think there are two docketNumbers being passed in and it isn't clear without reading the code which one wins.